script/hooks/pre-commit

```diff
@@ -9,16 +9,8 @@ if [ -z "$(IS_MERGING)" ]; then
   git stash push --quiet --include-untracked --keep-index
 fi
 
-# Checks
-npm run license-check
-npm run lint
-npm run transpile
-npm run vet
-
-if command -v gitleaks > /dev/null; then
-  gitleaks protect
-fi
```

Comment:
I haven't been using this for a while now and this project stopped using it in #1612 so I figured I might as well remove it.